services/app/src/components/avatarPicture/CustomAvatar.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Avatar } from 'antd';
+import DefaultAvatar from './DefaultAvatar';
+import { getAvatarPxSize } from '../../helpers/getAvatarSizes';
+
+const CustomAvatar = ({ size, imageUrl, id, isBorderNeeded }) => {
+  const [isAvatarError, setIsAvatarError] = useState(false);
+
+  if (isAvatarError) {
+    return (
+      <DefaultAvatar
+        size={size}
+        id={id}
+        isBorderNeeded={isBorderNeeded}
+      />  
+    )
+  }
+
+  return (
+    <Avatar 
+      size={getAvatarPxSize(size)}
```

Comment:
why this `size` property is wrapped in `getAvatarPxSize` while the `DefaultAvatar` `size` property is not?

Reply:
If you are talking about this particular file, then it is not wrapped in ```getAvatarPxSize``` because ```DefaultAvatar``` takes size as a prop(small, medium, ect). And then in ```DefaultAvatar.jsx``` it takes that argument and calls it with ```getAvatarPxSize```.